### PR TITLE
#2 fix typos in JCIP notes

### PR DESCRIPTION
**Pull request by gavvvr**
I've read JCIP recently and found your summary. While reading it to refresh the content of the book I found many typos (and a couple of mistakes) so I decided to make a PR

### CODE REVIEW

File: java/java-concurrency-in-practice/chapter-11/README.md [hunk 1]
```diff
@@ -373,6 +373,6 @@ Many logging libraries are thin wrappers around `println` utilities. This is an
 An alternative is to have a dedicated logging thread which solely logs incoming requests. All other threads append their messages to a blocking queue & continue their work.
 This is often more performant than the former approach as it involves less context switching.
 
-When multiple threads are trying to write to stdin, they are all blocked on I/O & context switching occurs, waiting for the bounded resource.
+When multiple threads are trying to write to stdout, they are all blocked on I/O & context switching occurs, waiting for the bounded resource.
```
Comment (gavvvr) [inline java/java-concurrency-in-practice/chapter-11/README.md:376]: I am sure you meant **stdout** here

File: java/java-concurrency-in-practice/chapter-16/README.md [hunk 2]
```diff
@@ -220,15 +220,15 @@ public class DoubleCheckedLocking {
...
-If it is observed to not be initialized, the synchronized block guarantees that if the object was indeed intialized in another thread, it won't be reinitialized again.
+If it is observed to not be initialized, the synchronized block guarantees that if the object was indeed initialized in another thread, it won't be reinitialized again.
 
 However, there is still a problem with this code as the worst-case scenario is that a thread would observe that an object's reference is initialized, but its fields are not (fully) initialized.
 
 This can be fixed by making the variable `volatile`, which doesn't incur very large performance costs.
 However, the necessity for this technique has long passed and it shouldn't be used in modern codebases.
 
 ## Initialization safety
-The final fields of a **properly constructed** object will not be observed in an inconsistent state even if an object is not safely published.
```
Comment (gavvvr) [inline java/java-concurrency-in-practice/chapter-16/README.md:231]: This statement should have been obviously an opposite

**Review by gavvvr — COMMENTED** [review]
Added notes on mistakes, all the rest is typos

File: java/java-concurrency-in-practice/chapter-16/README.md (hunk 2 above)
Comment (gavvvr) [reply java/java-concurrency-in-practice/chapter-16/README.md]: I reverted this change, sorry, it was correct

**Comment by gavvvr** [comment]
@preslavmihaylov could you please review it and merge? I basically fixed typos here.
I do the cleanup of my github and want to remove my fork.

Thanks for your notes. I used them for additional recap after reading JCIP from cover to cover in 2021.

**Review by preslavmihaylov — APPROVED** [review]
Thanks for your contribution and sorry for the looong delay, somehow I missed this PR!

It's great.

**Pull request merged**